edgexfoundry/edgex-go
# feat: generate spdx sbom and add Makefile target for sbom generation

Closes: #4173

Signed-off-by: Jim Wang <yutsung.jim.wang@intel.com>







**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x ] I am not introducing a breaking change (if you are, flag in conventional commit message with `BREAKING CHANGE:` describing the break)
- [x ] I have fully tested (add details below) this the new feature or bug fix (if not, why?)
  <link to docs PR>

## Testing Instructions

- git clone this branch
- run `make sbom` and it should run successfully and produce sbom under the directory sbom\


## New Dependency Instructions (If applicable)

spdx docker container used for generating spdx-sbom:     https://github.com/opensbom-generator/spdx-sbom-generator
